<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar {
    background-color: #fdfdfd;
    border: 1px solid #cccccc;
}

QMenuBar::item {
    spacing: 4px;
    padding: 6px 14px;
    background: transparent;
    color: #333;
}

QMenuBar::item:selected {
    background: #cce0ff;
    color: #000;
    border-radius: 6px;
}

QMenu {
    background-color: #ffffff;
    border: 1px solid #c0c0c0;
    padding: 4px;
    border-radius: 6px;
}

QMenu::item {
    padding: 6px 18px;
    background-color: transparent;
    color: #333;
    border-radius: 4px;
}

QMenu::item:selected {
    background-color: #e6f0ff;
    color: #000;
}
QWidget {
    background-color: #fffff0;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_agree">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #333333;
    border: 2px solid #0078d7;         /* синий акцент */
    border-radius: 8px;
    padding: 6px 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #e6f0ff;         /* светло-синий при наведении */
}

QPushButton:pressed {
    background-color: #cce0ff;         /* темнее при нажатии */
}

QPushButton:disabled {
    background-color: #f0f0f0;
    color: #a0a0a0;
    border: 1px solid #cccccc;
}
#pushButton_duplicate {
    background-color: #f9f9f9;
    border: 2px solid #28a745;   /* зелёный */
    border-radius: 8px;
    padding: 6px 14px;
}
#pushButton_duplicate:hover {
    background-color: #e6ffe9;
}
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_buildings">
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>831</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_for_checkboxes"/>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>62</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #5c4033;          /* тёмно-серый текст */
    font-size: 16px;
    font-weight: 500;
    padding: 1px 2px;
}
</string>
     </property>
     <property name="text">
      <string>Выберите тип здания или сооружения</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
